examples/chapter1/CMakeLists.txt

```diff
@@ -23,66 +23,117 @@ PROJECT(${PROJ_NAME})
 
 # Include directories with headers
 #---------------------------------------------------#
-INCLUDE_DIRECTORIES( ${CMAKE_CURRENT_SOURCE_DIR}/include )
-INCLUDE_DIRECTORIES( ${KLEPSYDRA_INCLUDE_DIRS} )
-
 ADD_EXECUTABLE(${PROJ_NAME}_1 ${${PROJ_NAME}_HEADERS} src/example1.cpp )
 
 TARGET_LINK_LIBRARIES(${PROJ_NAME}_1
+  PUBLIC
   ${KLEPSYDRA_CORE_LIBRARIES}
-  ${CMAKE_THREAD_LIBS_INIT}
   atomic
   )
 
+TARGET_INCLUDE_DIRECTORIES(${PROJ_NAME}_1 
+  PRIVATE
+  ${CMAKE_CURRENT_SOURCE_DIR}/include 
+  ${KLEPSYDRA_INCLUDE_DIRS}
```

Comment:
Have you tried deleting this line? It should not be necessary for compilation.

Reply:
Yes, I tried everything you told me but It didn't compile withou this line

Reply:
After the kpsr-core update in my PC this works . So, done